### ISSUE TITLE
weird two domains encoding?

### ISSUE THREAD

**Issue by FadeMind** [StevenBlack/hosts#465]
@StevenBlack  FYI. Any idea? :> 

Windows 10. 
Python 3.6.4.

Weird results:
```
==>0.0.0.0 130.211.230.53<==
==>0.0.0.0 É˘oogle.com<==
```

Full log
```
Microsoft Windows [Version 10.0.16299.192]
(c) 2017 Microsoft Corporation. Wszelkie prawa zastrzeżone.

C:\WINDOWS\system32>cd c:\Users\fademind\hosts\

c:\Users\fademind\hosts>updateHostsWindows.bat
Updating source data\adaway.org from https://raw.githubusercontent.com/AdAway/adaway.github.io/master/hosts.txt
Updating source data\add.2o7Net from https://raw.githubusercontent.com/FadeMind/hosts.extras/master/add.2o7Net/hosts
Updating source data\add.Dead from https://raw.githubusercontent.com/FadeMind/hosts.extras/master/add.Dead/hosts
Updating source data\add.Risk from https://raw.githubusercontent.com/FadeMind/hosts.extras/master/add.Risk/hosts
Updating source data\add.Spam from https://raw.githubusercontent.com/FadeMind/hosts.extras/master/add.Spam/hosts
Updating source data\Badd-Boyz-Hosts from https://raw.githubusercontent.com/mitchellkrogza/Badd-Boyz-Hosts/master/hosts
Updating source data\KADhosts from https://raw.githubusercontent.com/azet12/KADhosts/master/KADhosts.txt
Updating source data\malwaredomainlist.com from http://www.malwaredomainlist.com/hostslist/hosts.txt
Updating source data\mvps.org from http://winhelp2002.mvps.org/hosts.txt
Updating source data\someonewhocares.org from http://someonewhocares.org/hosts/zero/hosts
Updating source data\SpotifyAds from https://raw.githubusercontent.com/FadeMind/hosts.extras/master/SpotifyAds/hosts
Updating source data\StevenBlack from https://raw.githubusercontent.com/StevenBlack/hosts/master/data/StevenBlack/hosts
Updating source data\tyzbit from https://raw.githubusercontent.com/tyzbit/hosts/master/data/tyzbit/hosts
Updating source data\UncheckyAds from https://raw.githubusercontent.com/FadeMind/hosts.extras/master/UncheckyAds/hosts
Updating source data\yoyo.org from https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&mimetype=plaintext&useip=0.0.0.0
Updating source extensions\fakenews from https://raw.githubusercontent.com/marktron/fakenews/master/fakenews
Updating source extensions\gambling from https://raw.githubusercontent.com/Sinfonietta/hostfiles/master/gambling-hosts
Updating source extensions\porn\clefspeare13 from https://raw.githubusercontent.com/Clefspeare13/pornhosts/master/0.0.0.0/hosts
Updating source extensions\porn\sinfonietta from https://raw.githubusercontent.com/Sinfonietta/hostfiles/master/pornography-hosts
Updating source extensions\porn\sinfonietta-snuff from https://raw.githubusercontent.com/Sinfonietta/hostfiles/master/snuff-hosts
Updating source extensions\social from https://raw.githubusercontent.com/Sinfonietta/hostfiles/master/social-hosts
==>0.0.0.0 130.211.230.53<==
==>0.0.0.0 É˘oogle.com<==
Success! The hosts file has been saved in folder
It contains 42,977 unique entries.
        1 file(s) copied.

Windows IP Configuration

Successfully flushed the DNS Resolver Cache.
```

Regards

**Comment by ScriptTiger** [comment]
It's coming from someonewhocares.org, but the filters built into the script should scrub it out of the resultant hosts file because it's an illegal format, if you will. At least I believe so, yes? I had documented it before in the past and made my own filters for Windows, but I took it out because I thought it was solved in the repo's python script. Have you looked at your hosts file and seen if they made it into it?
